build/dockerfiles/rhel.Dockerfile

```diff
@@ -27,8 +27,8 @@ RUN /dashboard/.yarn/releases/yarn-*.cjs compile
 # https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhscl/httpd-24-rhel7
 FROM registry.access.redhat.com/rhscl/httpd-24-rhel7:2.4-143
 # DOWNSTREAM: use RHEL8/httpd
-# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhel8/httpd-24
-# FROM registry.redhat.io/rhel8/httpd-24:1-130.1618434979
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/httpd-24
+# FROM registry.redhat.io/ubi8/httpd-24:1-140
```

Comment:
registry is registry.access.redhat.com no ?

Reply:
yes. see update